### PR TITLE
add-four-out-of-14-new-layers

### PR DESCRIPTION
**Pull request by ana-berger**
Layers in this commit:

- dwm_sofs_surfaceprop_rt_data
- dwm_sofs_surfaceprop_rt_map
- dwm_ts_timeseries_data
- dwm_ts_timeseries_map

Ten more to come

### CODE REVIEW

**Comment by ana-berger** [comment]
Layers added in the commit [5c726d7](https://github.com/aodn/geoserver-config/pull/673/commits/5c726d7772218660c4c716edf4e18e4e6bc28678) all from the anmn_metadata store :
- dwm_all_map
- dwm_currents_map
- dwm_sots_dm_map
- dwm_sots_rt_map

Six more to come

**Comment by ana-berger** [comment]
Layers (and store) added in the commit [faa721c](https://github.com/aodn/geoserver-config/pull/673/commits/faa721c2050037e8080264e24ec79f46443aa4b1)
for the dwm_sofs_fl:
- dwm_sofs_surfaceflux_dm_data
- dwm_sofs_surfaceflux_dm_map
- dwm_sofs_surfaceflux_rt_data
- dwm_sofs_surfaceflux_rt_map
These layers were created with views with no data in it, as the harvester in talend is having issues

Two last ones missing (dwm_sofs_sp_dm_data/map)

File: workspaces/imos/JNDI_anmn_metadata/dwm_all_map/featuretype.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  UNIT[&quot;degree&quot;, 0.017453292519943295], 
+  AXIS[&quot;Geodetic longitude&quot;, EAST], 
+  AXIS[&quot;Geodetic latitude&quot;, NORTH], 
+  AUTHORITY[&quot;EPSG&quot;,&quot;4326&quot;]]</nativeCRS>
+  <srs>EPSG:4326</srs>
+  <nativeBoundingBox>
+    <minx>142.2334</minx>
+    <maxx>153.899</maxx>
+    <miny>-47.0227</miny>
+    <maxy>-27.326</maxy>
```
Comment (mhidas) [inline workspaces/imos/JNDI_anmn_metadata/dwm_all_map/featuretype.xml:35]: These bounding boxes should be set to -180 to 180, and -90 to 90 for all layers. Same for the `latLonBoundingBox` below.

File: workspaces/imos/JNDI_anmn_metadata/dwm_currents_map/featuretype.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  <title>dwm_currents_map</title>
+  <keywords>
+    <string>features</string>
+    <string>dwm_currents_map</string>
+  </keywords>
+  <metadataLinks>
+    <metadataLink>
+      <type>text/xml</type>
+      <metadataType>TC211</metadataType>
+      <content>https://catalogue-123.aodn.org.au/geonetwork/srv/eng/xml_iso19139.mcp?uuid=b355a5a5-e6ec-45c7-8506-103cf160431a&amp;styleSheet=xml_iso19139.mcp.xsl</content>
```
Comment (mhidas) [inline workspaces/imos/JNDI_anmn_metadata/dwm_currents_map/featuretype.xml:17]: I'm not sure this is absolutely necessary, but if we're going to include a metadata link, it should point to the correct portal collection record. E.g. for `dwm_currents_map`, it should be [Deep Water Moorings - Current velocity time-series](https://catalogue-imos.aodn.org.au/geonetwork/srv/api/records/f7e49e83-4579-413c-b092-2187dca8395d) (uuid=f7e49e83-4579-413c-b092-2187dca8395d)

**Review by mhidas — COMMENTED** [review]
A couple of comments, as discussed.

File: workspaces/imos/JNDI_anmn_metadata/dwm_all_map/featuretype.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<featureType>
+<featureType
```
Comment (mhidas) [inline workspaces/imos/JNDI_anmn_metadata/dwm_all_map/featuretype.xml:1]: Accidental edit? That '>' is needed!

Comment (ana-berger) [reply workspaces/imos/JNDI_anmn_metadata/dwm_all_map/featuretype.xml]: yes, I tried to paste the metadata UUID with `ctrl+C` and deleted the `>` accidentally, fixed now

**Pull request closed without merging**